src/api/plugins/video/warren_video/models.py

```diff
@@ -1 +1,26 @@
 """Defines the schema of what is returned inside the API responses."""
+
+from pydantic import BaseModel, validator
+
+Seconds = int | None
+Percent = float | None
+
+
+class Info(BaseModel):
+    name: str | None
```

Comment:
```suggestion
    name: Union[str, None]
```
Available only from Python version 3.10+

Reply:
good catch, fixed up !